lib/worldwide/address.rb

```diff
@@ -156,7 +163,14 @@ def concatenate_address1
     end
 
     def concatenate_address2
-      additional_fields = region.combined_address_format["address2"] || []
+      detected_scripts = Worldwide::Scripts.identify(text: neighborhood).map(&:to_s)
```

Comment:
Right now in ruby we're only doing this check on `neighborhood`, but in typescript we're doing it on any field defined in `address2`.

Reply:
Seeing as both line2 and neighborhood are optional, I think we should consider running the identification on both fields, perhaps prefering the latter? Unit numbers may not be the best predictors, but they could act as tiebreakers.